### PR TITLE
setup: Jest for client and server

### LINKED ISSUE #14: Setup Tests

### PR DESCRIPTION
**Pull request by vineetvk01**
command: yarn test 
Test will be run for both client and server. 

Closes #14

### CODE REVIEW

**Review by ViralRuparel — APPROVED** [review]
Ok to go, proceed with further reviews.

**Comment by vineetvk01** [comment]
Please provide review as the conflits are resolved. @ViralRuparel  @rishabh1403

**Review by ViralRuparel — APPROVED** [review]
LGTM proceed with further review.

File: client/src/__tests__/index.test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import { shallow } from 'enzyme';
+
+import Index from '../views/index';
+
+describe('Check Index View',()=>{
+    it('should render the page',()=>{
+        const component = shallow(<Index />);
+        expect(component.find('h1').text()).toBe('WeConnect');
+    })
+})
```
Comment (rishabh1403) [inline client/src/__tests__/index.test.js]: add new line at end of file, same applicable in all files

**Review by rishabh1403 — APPROVED** [review]
minor changes

**Pull request merged**